worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -399,6 +400,8 @@ private void updateJobProgress(ContractData contractData, ProgressTrackerUpdate
         jobChannelService.sendUpdate(jobUuid, JobMeasure.PATIENT_REQUESTS_ERRORED,
                 updateTracker.getPatientFailureCount());
 
+        jobChannelService.sendUpdate(jobUuid, JobMeasure.PATIENTS_WITH_EOBS, updateTracker.getPatientWithEobCount());
```

Comment:
small, Have updateTracker.getPatientWithEobCount()); on the next line like the rest.